script.js

```diff
@@ -23,4 +23,116 @@ closeMenuTwo.addEventListener('click', () => {
 
 closeMenuThree.addEventListener('click', () => {
   burgerMenu.classList.toggle('menu_active');
-});
\ No newline at end of file
+});
+
+const cardOne = [{
+  name: 'Multi Post Stories',
+  nameTwo: 'Keeping track of hundreds  of components website',
+  technologies: ['html', 'Bootstrap', 'Ruby on rails'],
+  image: 'images/SnapshootPortfolio.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  liveVersion: 'see live',
+  source: 'see source',
+}];
```

Comment:
- [ ] Kindly as the previous review asked, you here have an array of one object, data for one project, but you should have data for 6-7 projects that's mean you have to have 6-7 different objects everyone has data for a different project.

Like 
```
const projects = [{
    name: "Project one ",
    description: "Project one des",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}, 
{
    name: "Project two",
    description: "Project two des .",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}, 
{
    name: "Project three",
    description: "Project three des .",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}, 
{and the same for project four}, 
{and the same for project five}, 
{and the same for project six}]
```

And use this data to display it in the modal.

Reply:
Thank you @keroloussamy  I will add more objects into the array.